recommend/policyRules.go

```diff
@@ -40,34 +36,25 @@ type Description struct {
 
 var policyRules []MatchSpec
 
-//go:embed yaml/rules.yaml
-var policyRulesYAML []byte
-
-func updateRulesYAML(yamlFile []byte) string {
-	policyRules = []MatchSpec{}
-	if len(yamlFile) < 30 {
-		yamlFile = policyRulesYAML
-	}
+func updateRulesYAML(yamlFile []byte, policyRules *[]MatchSpec) (string, error) {
```

Comment:
This PR removes dependency on `yaml/rules.yaml` altogether. Can we delete the file from the repo?